ResourceStatusSystem/DB/ResourceManagementDB.py

```diff
@@ -456,6 +514,8 @@ def select( self, table, params ):
             select = select.filter(column_a == columnValue)
           elif isinstance(columnValue, datetime.datetime): #FIXME: is it correct/enough? (should check also below)
             select = select.filter(column_a == columnValue)
+          elif isinstance(columnValue, bool):
+            select = select.filter(column_a == columnValue)
```

Comment:
I think this can be simplified:

```python
if isinstance(columnValue, (list, tuple)):
             select = select.filter(column_a.in_(list(columnValue)))
elif isinstance(columnValue, (basestring, datetime.datetime,bool)):
  select = select.filter( column_a == columnValue )
else:
 ...
```